### DISCUSSION TITLE
small website UI suggestion

### DISCUSSION THREAD

**Discussion by zabbazebra** [ppy/osu#27103] [Ideas]
Its not super urgent, but i feel it would be cleaner and definitely easier to read if we changed how DA changes were listed.

When multiple aspects are changed using DA, it looks a bit messy:

![image](https://github.com/ppy/osu/assets/157727110/67d334db-09d7-4ef9-8f90-264b6f7c1a00)

I think it would be nicer as seen here in my rough sketch:

![image](https://github.com/ppy/osu/assets/157727110/75d7725a-3ba8-4109-9b24-002716adb207)

**Comment by mixalisnew** [comment]
That actually looks really clean if it can be dons it would be cool

**Comment by Joehuu** [comment]
Lazer uses the acronyms. Unsure which is better, but should be consistent.

![osu!_EalE9qP69B](https://github.com/ppy/osu/assets/35318437/6155d2c6-43c9-4834-8d1e-bc0f76c13c0d)

**Reply by NafeeJ** [reply]
i think acronyms are better. my guess is that if you're using diff adjust you probably know them anyway. therefore no point to the extra visual bloat on the websites implementation

**Reply by bdach** [reply]
Acronyms are not being used because it's a pain to ferry the text formatting from lazer to web.

Not sure of the correct way to address this.

**Comment by OwenCMYK** [comment]
I don't have a rough draft, but what if each of the stats you changed displayed a mod-like icon. For example instead of it saying "DA" it just says "CS 1.5", or "HP 2.2". And if you changed multiple it could display them separately. That might not be the best solution but I just thought I'd throw that out there.

**Reply by bdach** [reply]
https://github.com/ppy/osu/discussions/24961

**Reply by OwenCMYK** [reply]
Thank you bdach!
